src/main/asciidoc/_chapters/slow_log_responses_from_systable.adoc

```diff
@@ -0,0 +1,116 @@
+////
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+////
+
+[[slow_log_responses_from_systable]]
+==== Get Slow/Large Response Logs from System table hbase:slowlog
+
+The above section provides details about Admin APIs:
+
+* get_slowlog_responses
+* get_largelog_responses
+* clear_slowlog_responses
+
+All of the above APIs access online in-memory ring buffers from
+individual RegionServers and accumulate logs from ring buffers to display
+to end user. However, they can only be used for an ongoing slow/large RPC calls
+and user can retrieve ongoing issues with RPC logs. After RegionServer is
```

Comment:
BTW these logs in ring buffer will have any timestamp ? (When the slow/large RPC happened?)

Reply:
Yes, they have timestamp of when they occured. One sample:
```
hbase(main):004:0* get_largelog_responses '*'
2020-05-07 19:05:10,882 WARN  [main] util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
Retrieved LargeLog Responses from RegionServers
{
  "startTime": 1588858410089,
  "processingTime": 1,
  "queueTime": 0,
  "responseSize": 2,
  "clientAddress": "172.20.10.2:57523",
  "serverClass": "HRegionServer",
  "methodName": "Mutate",
  "callDetails": "Mutate(org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$MutateRequest)",
  "param": "region\u003d hbase:meta,,1",
  "userName": "vjasani"
}
{
  "startTime": 1588858409968,
  "processingTime": 1,
  "queueTime": 0,
  "responseSize": 4,
  "clientAddress": "172.20.10.2:57523",
  "serverClass": "HRegionServer",
  "methodName": "Get",
  "callDetails": "Get(org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$GetRequest)",
  "param": "region\u003d hbase:meta,,1, row\u003d hbase:slowlog",
  "userName": "vjasani"
}
{
  "startTime": 1588858409945,
  "processingTime": 1,
  "queueTime": 0,
  "responseSize": 4,
  "clientAddress": "172.20.10.2:57523",
  "serverClass": "HRegionServer",
  "methodName": "Get",
  "callDetails": "Get(org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$GetRequest)",
  "param": "region\u003d hbase:meta,,1, row\u003d hbase:slowlog",
  "userName": "vjasani"
}

```